LEAF_Request_Portal/admin/templates/mod_svcChief.tpl

```diff
@@ -166,57 +166,94 @@ function removeUser(groupID, userID) {
 }
 
 function initiateWidget(serviceID, serviceName) {
```

Comment:
Are default values needed here?

Reply:
I don't believe so. This event only triggers when clicking on the group card. So there will always be some value in these parameters.